cpp/src/prims/detail/nbr_intersection.cuh

```diff
@@ -601,6 +750,8 @@ nbr_intersection(raft::handle_t const& handle,
   std::optional<rmm::device_uvector<size_t>> major_nbr_offsets{std::nullopt};
   std::optional<rmm::device_uvector<vertex_t>> major_nbr_indices{std::nullopt};
 
+  std::optional<rmm::device_uvector<edge_property_value_t>> major_nbr_properties{std::nullopt};
```

Comment:
Isn't this an optional buffer? Better be consistent and use the same mechanism to handle all optional buffers and spans.